src/howitz/templates/responses/502.html

```diff
@@ -0,0 +1,11 @@
+{% block content %}
+    <main
+            class="py-8 px-4 mx-auto max-w-screen-xl lg:py-16 lg:px-6 bg-slate-900 text-sky-200/80"
+    >
+        <section class="mx-auto max-w-screen-sm text-center">
+            <h1 class="mb-4 text-7xl tracking-tight font-extrabold lg:text-9xl">502</h1>
+            <p class="mb-4 text-3xl tracking-tight font-bold md:text-4xl">Bad Gateway</p>
+            <p class="mb-4 text-lg font-light text-teal-300">{{ err_msg }}</p>
+        </section>
+    </main>
+{% endblock %}
```

Comment:
Polish: mention that this is a problem on the server, not necessarily a problem with howitz code.